src/content/concepts/index.md

```diff
@@ -59,20 +62,18 @@ In the example above, we use the `output.filename` and the `output.path` propert
 
 T> You may see the term **emitted** or **emit** used throughout our documentation and [plugin API](/api/plugins). This is a fancy term for 'produced' or 'discharged'.
 
-The `output` property has [many more configurable features](/configuration/output), but let's spend some time understanding some of the most common use cases for the `output` property.
-
-[Learn more!](/concepts/output)
+The `output` property has [many more configurable features](/configuration/output) and if you like to know more about the concepts behind the `output` property, you can [read more in the concepts section](/concepts/output).
 
 
 ## Loaders
 
-The goal is to have all of the assets in your project be **webpack's** concern and not the browser's (though, to be clear, this doesn't mean that they all have to be bundled together). webpack treats [every file (.css, .html, .scss, .jpg, etc.) as a module](/concepts/modules). However, webpack itself **only understands JavaScript**.
+*Loaders* enable webpack to process more than just JavaScript files (webpack itself only understands JavaScript). They make sure that you can leverage webpack's bundling abilities for files that aren't JavaScript by treating them as [webpack modules](/concepts/modules).
 
-**Loaders in webpack _transform these files into modules_ as they are added to your dependency graph.**
+In short: **Loaders in webpack _transform non JavaScript files into modules_ as they are added to your dependency graph.**
```

Comment:
Not sure if we should keep this whole sentence bold. Also maybe we could rephrase to something like:

> Essentially, webpack loaders transform all types of files into modules that included in your application's dependency graph.

Reply:
Also, as you've already started doing, I think we can lowercase "Loaders" unless it's used at the beginning of a sentence.